src/components/PopoverContainer/PopoverContainer.module.css

```diff
@@ -10,4 +12,8 @@
   overflow: auto;
   box-shadow: var(--eds-box-shadow-md);
   background-color: var(--eds-theme-color-background-neutral-default);
+
+  &:focus-visible {
```

Comment:
by applying focus rings to the shared component, any future components we use can make use of this via the `as` prop. All Headless UI components work like this, so they get proper focus ring behavior thru this now